src/ess/spectroscopy/indirect/workflow.py

```diff
@@ -919,9 +989,9 @@ def component_names(
 
 def bifrost(
     filename: Filename,
+    tof_lookup_table: TimeOfFlightLookupTable,
```

Comment:
In the future, this could just become an instrument configuration, like [we did for Dream](https://github.com/scipp/essdiffraction/pull/136)